covidapi/api/static/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>COVID-19 Python API</title>
+</head>
+<body>
+<a href="https://github.com/andreagrandi/covid-api"><img width="149" height="149" src="https://github.blog/wp-content/uploads/2008/12/forkme_left_red_aa0000.png?resize=149%2C149" class="attachment-full size-full" alt="Fork project on GitHub" data-recalc-dims="1"></a>
```

Comment:
These image tags seem to come with some existing attributes and classes which seems a bit strange if there's no CSS/Javascript that references them - I'm assuming it looks the same if we strip these out?

Reply:
These attributes:
- `class="attachment-full size-full"` (and `class="middle"` below)
-  `data-recalc-dims="1"`

Reply:
Resolved: https://github.com/shiv-io/covid-api/commit/a08df94b13749f888227e04a77bb1e2b3085b148